crime-means-assessment/src/main/java/uk/gov/justice/laa/crime/meansassessment/builder/MaatCourtDataAssessmentBuilder.java

```diff
@@ -12,11 +15,13 @@
 import uk.gov.justice.laa.crime.meansassessment.dto.MeansAssessmentRequestDTO;
 import uk.gov.justice.laa.crime.meansassessment.staticdata.entity.AssessmentCriteriaEntity;
 
+import java.util.List;
+
 import static java.util.Optional.ofNullable;
 
+@Slf4j
```

Comment:
I dont see any logging output in this file, I guess has this been left in after adding logging during development?

Reply:
Done.